api/managementpb/dbaas/components.proto

```diff
@@ -102,6 +103,32 @@ message ChangePXCComponentsRequest {
 
 message ChangePXCComponentsResponse {}
 
+message InstallOperatorRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // operator_type tells what operator we are interested in updating.
+  string operator_type = 2 [
```

Comment:
Can we use enum here?

Reply:
@BupycHuk Sorry, did not notice it until now. It would require changes in FE, too and Tiago is on vacation. Can we add this later?

Reply:
I'm ok if it will be fixed before 2.21.0 RC

Reply:
Please create tech debt task and set fix version 2.21

Reply:
https://jira.percona.com/browse/PMM-8532